MergerLogic/Monitoring/OpenTelemetryFormattedConsoleExporter.cs

```diff
@@ -29,8 +29,9 @@ private string MCTextFormat(LogRecord record)
             var resource = this.ParseResource();
             var serviceName = this.GetResourceAttribute(resource, SERVICE_NAME_ATTRIBUTE, "unknown_service");
             var serviceVersion = this.GetResourceAttribute(resource, SERVICE_VERSION_ATTRIBUTE, "unknown_version");
-
-            return $"[{this.FormatTime(record.Timestamp)}] [{record.LogLevel}] [{serviceName}] [{serviceVersion}] [{record.CategoryName}] {record.State}";
+            var exception = record.Exception != null ? $"[{record.Exception}]" : string.Empty;
+            
+            return $"[{this.FormatTime(record.Timestamp)}] [{record.LogLevel}] [{serviceName}] [{serviceVersion}] [{record.CategoryName}] {record.State} {exception}";
```

Comment:
put the space inside the "exception" variable only if needed
return $"[{this.FormatTime(record.Timestamp)}] [{record.LogLevel}] [{serviceName}] [{serviceVersion}] [{record.CategoryName}] {record.State}{exception}";